tests/test_frontend_vgg.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+import torch
+from torch import nn
+
+from i6_models.parts.frontend import (
+    VGG4LayerActFrontendV1,
+    VGG4LayerActFrontendV1Config,
+)
+
+
+def test_conformer_vgg_layer_act_frontend_v1():
+    torch.manual_seed(42)
+
+    def get_output_shape(
+        batch,
+        time,
+        time_padding,
+        features,
+        conv1_channels,
+        conv2_channels,
+        conv3_channels,
+        conv4_channels,
+        pool1_red,
+        pool2_red,
+        in_dim,
+        out_dim,
+    ):
+        data_input = torch.randn(batch, time, features)
+        data_input = torch.cat((data_input, torch.zeros(batch, time_padding, features)), dim=1)
+        data_input = torch.cat((data_input, torch.randn(batch, time + time_padding, features)), dim=0)
+
+        sequence_mask = torch.ones(batch, time)
+        sequence_mask = torch.cat((sequence_mask, torch.zeros(batch, time_padding)), dim=1)
+        sequence_mask = torch.cat((sequence_mask, torch.ones(batch, time + time_padding)), dim=0)
+        sequence_mask = sequence_mask.bool()
```

Comment:
I don't really understand this arbitrary creation of the seq mask. Is this on purpose? Why don't you pass it as an argument, just like the `data_input`?

Reply:
I am not sure why this matters for the tests here. Can you elaborate?

Reply:
The code looks like you are trying to create a specific sequence mask with some meaning. But I fail to understand the meaning, the logic of the code. But now if I get you right, there is no real meaning at all, it is supposed to be just random. But if that is the case, I would just more explicitly make it random. I would maybe create random seq lens first and then create the seq mask based on that, such that the seq mask is reasonable as we would usually have it.

Reply:
yes, there is no deeper meaning.
ok. I can do that